v3/src/components/slider/slider-component.tsx

```diff
@@ -70,17 +68,12 @@ export const SliderComponent = observer(({sliderModel} : IProps) => {
   }
 
   const incrementSliderValue = () => {
```

Comment:
@kswenson now that I look at the other changes, should `incrementSliderValue` be a method in the model?